porter.yaml

```diff
@@ -123,6 +132,7 @@ upgrade:
         - manifests/
       wait: true
       validate: false
+      namespace: "{{ bundle.parameters.namespace }}"
```

Comment:
need to add this as part of qliksense as well in upgrade:

Reply:
resolved.